bin/pub_release.dart

```diff
@@ -20,7 +20,12 @@ void main(List<String> args) {
   parser.addOption('line', abbr: 'l', help: 'Specifies');
 
   parser.addCommand('help');
-  final results = parser.parse(args);
+  late final ArgResults results;
+  try {
+    results = parser.parse(args);
+  } catch (_) {
+    results = parser.parse(['help']);
```

Comment:
I think it would be better to print the exception and then call:

showUsage(parser);
    exit(-1);

Reply:
Why would you print an unreadable exception for something the user doesn't have control over?

Reply:
A bad error is always better than no error :)